src/components/dashboard/FaceVerification/components/Instructions.js

```diff
@@ -60,6 +73,53 @@ const Instructions = ({ styles, onDismiss = noop }) => (
   </Wrapper>
 )
 
+const InstructionsB = ({ styles, onDismiss = noop }) => (
+  <Wrapper>
+    <Section style={styles.topContainerB} grow>
+      <View style={styles.mainContentB}>
+        <ImageBackground
+          source={require('../../../../assets/FaceVerification/FVPortrait.png')}
```

Comment:
why dont use you the "portrait" you iimported. also you didnt use srcset.
@serdiukov-o-nordwhale i think user barely sees difference. i suggest to use 2x for web and the import 'portrait' for native which should import the correct  1x/2x/3x

Reply:
how do I pick the 2X resolution only for web?

Reply:
done